modules/olm-deleting-operators-from-a-cluster-using-web-console.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * applications/operators/olm-adding-operators-to-cluster.adoc
+
+[id='olm-deleting-operators-from-a-cluster-using-web-console-{context}']
+= Deleting Operators from a cluster using the web console
+
+To delete an installed Operator from the selected namespace through the Web console, follow these steps:
+
+.Procedure
+
+. Navigate in the web console to the *Catalog → Operator Management* page.
+. Select the *Operator Subscriptions* tab to see the list of installed Operators
+for the namespace.
+. Find the Operator you want to delete (in this example, jaeger) and click
+the three-dot drop-down box in far right column.
++
+image::olm-operator-delete.png[]
+
+. Select *Remove Subscription*.
```

Comment:
I think this counts as a menu, so s/Select/cClick

Reply:
Done.